test/e2e/manifest/migration-windows.json

```diff
@@ -41,7 +41,7 @@
         "windowsProfile": {
             "adminUsername": "azureuser",
             "adminPassword": "replacepassword1234$",
-            "csiProxyURL": "https://acs-mirror.azureedge.net/csi-proxy/v1.0.2/binaries/csi-proxy-v1.0.2.tar.gz",
+            "csiProxyURL": "https://acs-mirror.azureedge.net/csi-proxy/v0.2.2/binaries/csi-proxy-v0.2.2.tar.gz",
```

Comment:
Odd, I wonder why this was changed to the pre GA version of CSI Proxy?

Reply:
I see that this change was made 3 months ago https://github.com/kubernetes-sigs/azuredisk-csi-driver/commit/e3d0c08f4d95baf4c9a6c5906f45e705567f6e13. Maybe I am missing something but I can't find a PR for this.